rectools/models/vector.py

```diff
@@ -157,87 +177,15 @@ def rank(
             num_threads=num_threads,
         )
 
-        if sorted_item_ids_to_recommend is not None:
-            ids = sorted_item_ids_to_recommend[ids]
+        if sorted_object_whitelist is not None:
+            ids = sorted_object_whitelist[ids]
 
         # filter neginf from implicit scores and apply norms for correct COSINE distance
```

Comment:
# filter neginf from implicit scores and compute correct COSINE and EUCLIDEAN scores

Reply:
fixed